main.go

```diff
@@ -166,209 +39,42 @@ func setupSignals(socketPath string) {
 	}()
 }
 
-// NewPlugin instantiates a new plugin
-func NewPlugin() (*Plugin, error) {
-	plugin := &Plugin{
-		pvs:     getPVs(),
-		Latpvs:  getLatPVs(),
-		Tputpvs: getTputPVs(),
-	}
-	return plugin, nil
-}
-
-func getValue(body []byte) (*Iops, error) {
-	storeBefore := new(Iops)
-	err := json.Unmarshal(body, &storeBefore)
+func main() {
+	// creating in-cluster config
+	config, err := rest.InClusterConfig()
 	if err != nil {
-		fmt.Println("whoops:")
+		log.Error(err)
 	}
-	return storeBefore, err
-}
 
-func getLatValue(body []byte) (*Iops, error) {
-	s1 := new(Iops)
-	err := json.Unmarshal(body, &s1)
+	// create clientset of kubernetes
+	metrics.ClientSet, err = kubernetes.NewForConfig(config)
```

Comment:
This is not right approach! If you're assigning value to variable in metrics package then have this call there only.